src/clocksi_interactive_tx_coord_fsm.erl

```diff
@@ -302,36 +304,14 @@ perform_update(Args, Updated_partitions, Transaction, Sender, ClientOps) ->
                                 lists:keyreplace(IndexNode, 1, Updated_partitions,
                                                  {IndexNode, [{Key, Type, DownstreamRecord} | WriteSet]})
                         end,
-                    case Sender of
```

Comment:
This was used before to replying to the client here, when issuing single operation updates.
Now the reply is done in the "receive logging responses" state.